pkgs/http/README.md

```diff
@@ -124,7 +124,7 @@ Some well supported implementations are:
 | Implementation | Supported Platforms | SDK | Caching | HTTP3/QUIC | Platform Native | 
 | -------------- | ------------------- | ----| ------- | ---------- | --------------- |
 | `package:http` — [`IOClient`][ioclient] | Android, iOS, Linux, macOS, Windows | Dart, Flutter | ❌ | ❌ | ❌ |
-| `package:http` — [`BrowserClient`][browserclient] | Web | Flutter | ― | ✅︎ | ✅︎ | Dart, Flutter |
+| `package:http` — [`BrowserClient`][browserclient] | Web | Dart, Flutter | ― | ✅︎ | ✅︎ | Dart, Flutter |
```

Comment:
The `BrowserClient` doesn't run on the Dart VM, it is Web and flutter-web only. I'm not 100% sure, but I think "Dart" here refers to the VM or AOT compiles?

Reply:
I was trying to indicate whether flutter is required or not. I was copying the notation used by pub e.g. 
<img width="168" alt="image" src="https://github.com/dart-lang/http/assets/8851008/43c9013a-96ec-4578-8698-8a380706386a">

That's for `package:fetch_client` - which is web-only

Reply:
I do have a separate column for the platform, which is "web" in the case of `BrowserClient`

Reply:
Ah I see, that makes sense.